TeadsSampleApp/Controllers/RootController/RootViewController.swift

```diff
@@ -57,7 +63,8 @@ class RootViewController: TeadsViewController {
             case .custom:
                 return PID.custom
             case .nativeDisplay:
-                return PID.nativeDisplay
+                return PID.directNativeDisplay
+            default: return ""
```

Comment:
```suggestion
            default: 
                return ""
```

Reply:
we will handle this in a future PR supporting swiftformat